k8s/k8s.go

```diff
@@ -240,17 +237,19 @@ func CmdAddK8s(args *skel.CmdArgs, conf utils.NetConf, nodename string, calicoCl
 
 		// Create the endpoint object and configure it.
 		endpoint = api.NewWorkloadEndpoint()
-		endpoint.Metadata.Name = args.IfName
-		endpoint.Metadata.Node = nodename
-		endpoint.Metadata.Orchestrator = orchestrator
-		endpoint.Metadata.Workload = workload
-		endpoint.Metadata.Labels = labels
+		endpoint.Name = epIDs.WEPName
```

Comment:
minor nit, but we're mixing up the Spec and Meta fields here - would be nice to put the Name and Namespace and Labels at the top and spec fields after.